pkg/tfbridge/tokens_test.go

```diff
@@ -87,6 +87,24 @@ func TestTokensSingleModule(t *testing.T) {
 	}, info.Resources)
 }
 
+func TestTokenWithModuleName(t *testing.T) {
+	info := tfbridge.ProviderInfo{
+		Name: "foo",
+		P: (&schema.Provider{
+			ResourcesMap: schema.ResourceMap{
+				"foo_index": nil,
```

Comment:
should we be testing some other names against this strategy eg. maybe the list fromt he previous test:
```
"foo_fizz_buzz":       {Tok: "foo:index:FizzBuzz"},
"foo_bar_hello_world": {Tok: "foo:index:BarHelloPulumi"},
"foo_bar":             {Tok: "foo:index:Bar"},
```

Reply:
This strategy is pretty well tested in:

https://github.com/pulumi/pulumi-terraform-bridge/blob/610224f512cdc81b79ef771f36ba6ac9e9d3387c/pkg/tfbridge/tokens_test.go#L36